cristin-person/src/main/java/no/unit/nva/cristin/person/update/CristinPersonPatchJsonCreator.java

```diff
@@ -118,4 +122,21 @@ private void addEmployments() {
     private List<CristinPersonEmployment> employmentsToCristinFormat(List<Employment> employments) {
         return employments.stream().map(Employment::toCristinEmployment).collect(Collectors.toList());
     }
+
+    private void addKeywords() {
+        if (input.has(KEYWORDS) && !input.get(KEYWORDS).isNull()) {
+            var inputKeywords = input.get(KEYWORDS).toString();
+            var keywordsInCristinFormat =
+                attempt(() -> {
+                    var parsedInput = asList(OBJECT_MAPPER.readValue(inputKeywords, TypedValue[].class));
+                    return OBJECT_MAPPER.readTree(keywordsToCristinFormat(parsedInput).toString());
+                }).orElseThrow();
+            output.set(KEYWORDS, keywordsInCristinFormat);
+        }
+    }
```

Comment:
Do you have an example?

Reply:
I thought about something like this if we do not have to make null check, but since we ca possibly have null object, it is not relevant:

```
attempt(() -> input.get(KEYWORDS).toString())
            .map(string -> OBJECT_MAPPER.readValue(string, TypedValue[].class))
            .map(Arrays::asList)
            .map(this::keywordsToCristinFormat)
            .map(Object::toString)
            .map(OBJECT_MAPPER::readTree)
            .orElseThrow();
```

Reply:
We only need to do anything if the field keywords is present in the payload, so whatever we do we need to check that first anyways

Reply:
I will look into it a little more and see if it can be more declarative